### PR TITLE
Add Pepys v1.5.3

### PR DESCRIPTION
**Pull request by LukeBriggsDev**
- [x] Have you signed the [Contributor License Agreement](https://cla.opensource.microsoft.com/microsoft/winget-pkgs)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/microsoft/winget-pkgs/pulls) for the same manifest update/change?
- [x] Have you [validated](https://github.com/microsoft/winget-pkgs/blob/master/AUTHORING_MANIFESTS.md#validation) your manifest locally with `winget validate --manifest <path>`? 
- [x] Have you tested your manifest locally with `winget install --manifest <path>`?
- [x] Does your manifest conform to the [1.0 schema](https://github.com/microsoft/winget-cli/blob/master/doc/ManifestSpecv1.0.md)?

Note: `<path>` is the name of the directory containing the manifest you're submitting.

-----

###### Microsoft Reviewers: [Open in CodeFlow](https://portal.fabricbot.ms/api/codeflow?pullrequest=https://github.com/microsoft/winget-pkgs/pull/56368)

### CODE REVIEW

File: manifests/l/LukeBriggs/Pepys/1.5.3/LukeBriggs.Pepys.installer.yaml [hunk 1]
```diff
@@ -9,4 +9,4 @@ Installers:
   InstallerUrl: https://github.com/LukeBriggsDev/Pepys/releases/download/v1.5.3/pepys.1.5.3.win.x86_64.installer.exe
   InstallerSha256: 902ce9565d6c815e580ea35045c1223c79f6a876ff2a9f89e5b9662d1593885a
 ManifestType: installer
-ManifestVersion: 1.0.0
\ No newline at end of file
+ManifestVersion: 1.0.1
```
Comment (ItzLevvie) [inline manifests/l/LukeBriggs/Pepys/1.5.3/LukeBriggs.Pepys.installer.yaml:12]: This key must have the value "1.0.0". The Microsoft community package repository validation pipelines also use this value to determine appropriate validation rules when evaluating this file.

File: manifests/l/LukeBriggs/Pepys/1.5.3/LukeBriggs.Pepys.locale.en-US.yaml [hunk 2]
```diff
@@ -17,5 +17,5 @@ ShortDescription: A straightforward markdown journal
 Description: Pepys is a journaling application that utilises markdown for writing and storing the journal entries
 Moniker: pepys
 ManifestType: defaultLocale
-ManifestVersion: 1.0.0
+ManifestVersion: 1.0.1
```
Comment (ItzLevvie) [inline manifests/l/LukeBriggs/Pepys/1.5.3/LukeBriggs.Pepys.locale.en-US.yaml:20]: This key must have the value "1.0.0". The Microsoft community package repository validation pipelines also use this value to determine appropriate validation rules when evaluating this file.

File: manifests/l/LukeBriggs/Pepys/1.5.3/LukeBriggs.Pepys.yaml [hunk 3]
```diff
@@ -5,5 +5,5 @@ PackageIdentifier: LukeBriggs.Pepys
 PackageVersion: 1.5.3
 DefaultLocale: en-US
 ManifestType: version
-ManifestVersion: 1.0.0
+ManifestVersion: 1.0.1
```
Comment (ItzLevvie) [inline manifests/l/LukeBriggs/Pepys/1.5.3/LukeBriggs.Pepys.yaml:8]: This key must have the value "1.0.0". The Microsoft community package repository validation pipelines also use this value to determine appropriate validation rules when evaluating this file.

**Review by ItzLevvie — CHANGES_REQUESTED** [review]

File: manifests/l/LukeBriggs/Pepys/1.5.3/LukeBriggs.Pepys.installer.yaml (hunk 1 above)
Comment (ItzLevvie) [reply manifests/l/LukeBriggs/Pepys/1.5.3/LukeBriggs.Pepys.installer.yaml:12]: Also, 1.0.1 schema doesn't exist.

We're at 1.1.0 now.

File: manifests/l/LukeBriggs/Pepys/1.5.3/LukeBriggs.Pepys.locale.en-US.yaml (hunk 2 above)
Comment (ItzLevvie) [reply manifests/l/LukeBriggs/Pepys/1.5.3/LukeBriggs.Pepys.locale.en-US.yaml:20]: Also, 1.0.1 schema doesn't exist.

We're at 1.1.0 now.

File: manifests/l/LukeBriggs/Pepys/1.5.3/LukeBriggs.Pepys.yaml (hunk 3 above)
Comment (ItzLevvie) [reply manifests/l/LukeBriggs/Pepys/1.5.3/LukeBriggs.Pepys.yaml:8]: Also, 1.0.1 schema doesn't exist.

We're at 1.1.0 now.

**Review by jedieaston — APPROVED** [review]

**Review by ghost — APPROVED** [review]

**Pull request merged**